packages/cli/src/commands/typescript/generate/handleGenerate.mts

```diff
@@ -26,6 +26,7 @@ import {
 import { createClientContext, createOpenApiRequest } from "@osdk/shared.net";
 import { consola } from "consola";
 import * as fs from "node:fs";
+import type {} from "@osdk/generator../../../../../generator/src/MinimalFs.js";
```

Comment:
Did you mean to import the MinimalFs type? Also should we export the type from `@osdk/generator`

Reply:
why didn't the autofix stuff get rid of this 🤦 . I was thinking that MinimalFs was a little weird to export as public API so I just duck-typed it below.

Reply:
It can make sense to export since it's an argument to the generator, but no need to do it now